src/components/Notifications/Notifications.tsx

```diff
@@ -304,6 +321,9 @@ class NotificationManager {
         });
         comm_socket.on("active_game", (game) => {
             delete this.boards_to_move_on[game.id];
+            if (game.phase === "finished") {
+                delete this.active_boards[game.id];
```

Comment:
I would remove the `if` and leave the `delete` to always `delete this.active_boards[game.id];` It seems more consistent with what we do for `boards_to_move_on`. If the game is supposed to be in active_boards, we readd it anyway.

Reply:
I had the same feeling, but in the forum thread anoek advised to write it this way, and this part of the code is working as desired. It is only rendering 0 moves while total games > 0 and boards_to_move_on = 0 that is currently not working properly.